src/plugins/intel_cpu/src/nodes/experimental_detectron_detection_output.cpp

```diff
@@ -310,7 +327,7 @@ void ExperimentalDetectronDetectionOutput::execute(dnnl::stream strm) {
     std::vector<int> detections_per_class(classes_num_, 0);
     int total_detections_num = 0;
 
-    for (int class_idx = 1; class_idx < classes_num_; ++class_idx) {
+    for (int class_idx = 0; class_idx < classes_num_; ++class_idx) {
```

Comment:
This change introduced a regression in e2e test (ticket: 87044),  I guess because this change adds background classes to the outputs, is there any solid reason for this change ?

Reply:
Hi @usstq, this change is to harmonize the ref implementation with the op [specification](https://docs.openvino.ai/nightly/openvino_docs_ops_detection_ExperimentalDetectronDetectionOutput_6.html)

Unfortunately in the spec I hanen't found any tip 
- that the 0th class is always a background class
- that the background class should always be removed

I guess this should be handled together with `class_agnostic_box_regression` attribute.
(But since this attr has been unhandled in the reference implementation, and since I hadn't understood the meaning of this attr at the time of the fix, after some discussion we decided to leave it as it is, unhandled.)

Reply:
@usstq please drop me a line to my email (in the github profile), I'll provide you with some more context

Reply:
The documentation of Detectron2 also confirms that in Detectron(not Detectron2) class 0 is assumed to be background and should be filtered :
https://detectron2.readthedocs.io/en/latest/notes/compatibility.html?highlight=background#compatibility-with-detectron-and-maskrcnn-benchmark

Reply:
PR https://github.com/openvinotoolkit/openvino/pull/12663 is submitted to fix this behavior